### DISCUSSION TITLE
How to set the bracket highlighter column line thickness in nordic config?

### DISCUSSION THREAD

**Discussion by narenkulk** [AlexvZyl/nordic.nvim#87] [Q&A] [answered]
![PNG image](https://github.com/AlexvZyl/nordic.nvim/assets/66332810/f82a6bf0-8c47-47df-ba5a-cf9ebdffd584)
Hello,
I really love this theme and wanted to know how I can have a thick bracket highlighter column(sorry i don't know whats it called) like the attached screenshot . 

Thanks,
Narendra

**Comment by AlexvZyl** [comment, marked as answer]
Hi!

The thickness of that line has nothing to do with nordic, but rather another plugin.  I am assuming you are either using indent-blankline, or a Neovim distro?

See [here](https://github.com/AlexvZyl/nvim/blob/724300e9f9cef5eafa3bfd76d0250399455b84fc/lua/alex/ui/indent-blankline.lua#L8C1-L9) for an example.
